Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -77,98 +80,109 @@ protected override void OnElementChanged (ElementChangedEventArgs<Map> e)
 			}
 
 			var map = NativeMap;
-			if (map != null) {
-				map.SetOnCameraChangeListener (this);
+			if (map != null)
+			{
+				map.SetOnCameraChangeListener(this);
 				NativeMap.InfoWindowClick += MapOnMarkerClick;
 
 				map.UiSettings.ZoomControlsEnabled = Map.HasZoomEnabled;
 				map.UiSettings.ZoomGesturesEnabled = Map.HasZoomEnabled;
 				map.UiSettings.ScrollGesturesEnabled = Map.HasScrollEnabled;
 				map.MyLocationEnabled = map.UiSettings.MyLocationButtonEnabled = Map.IsShowingUser;
-				SetMapType ();
+				SetMapType();
 			}
-			
-			MessagingCenter.Subscribe<Map, MapSpan> (this, MoveMessageName, OnMoveToRegionMessage, Map);
+
+			MessagingCenter.Subscribe<Map, MapSpan>(this, MoveMessageName, OnMoveToRegionMessage, Map);
 
 			var incc = Map.Pins as INotifyCollectionChanged;
 			if (incc != null)
 				incc.CollectionChanged += OnCollectionChanged;
 		}
 
-		void OnCollectionChanged (object sender, NotifyCollectionChangedEventArgs notifyCollectionChangedEventArgs)
+		void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs notifyCollectionChangedEventArgs)
 		{
-			switch (notifyCollectionChangedEventArgs.Action) {
-			case NotifyCollectionChangedAction.Add:
```

Comment:
Can I argue that this is the right way to ident a switch ? :P
otherwise, there are 2 level of indentation for a single condition...